docs/astro/src/content/docs/guide/tooling/figma-inspector.mdx

```diff
@@ -0,0 +1,88 @@
+## Setting up the Figma Inspector
+First of all, find and enable the "Figma to Slint" plugin in the Figma Plugin Manager.
+The inspector currently has 2 functions.  One is to inspect individual Figma objects for their properties, and the other is to export the variables defined in Figma so they are useable in Slint directly.
+
+## Key Features of Variable Export
+
+-   **Type Conversion:** Maps Figma variable types (Color, Float, String, Boolean) to corresponding Slint types (`brush`, `length`, `string`, `bool`).
+-   **Mode Support:** Translates Figma Modes into:
+    -   A Slint `enum` for mode selection (e.g., `MyCollectionMode { light, dark }`).
+    -   Collection-specific multi-mode convenience structs (e.g., `my_collection_mode2_brush { light: brush, dark: brush }`) for properties that vary by mode.
+    -   A `Scheme` struct (`MyCollection-Scheme`) representing the structure of variables.
+    -   A `Scheme-Mode` struct (`MyCollection-Scheme-Mode`) holding instances of the `Scheme` for each mode.
+    -   A global instance (`my_collection`) containing:
+        -   `mode`: An instance of `Scheme-Mode` holding the resolved values for each mode.
+        -   `current-mode`: An `in-out` property using the mode `enum` to control the active mode.
+        -   `current-scheme`: An `out` property dynamically selecting the correct scheme instance based on `current-mode`.
+        -   `current`: An `out` property mirroring the `Scheme` structure, providing direct access to the values of the `current-scheme`.
+-   **Alias Resolution:** Follows variable aliases (references) to find the underlying concrete value or target variable path.
+-   **Hierarchy Generation:** Interprets `/` in Figma variable names (e.g., `colors/background/primary`) to generate nested Slint structs (e.g., `struct Colors_Background { primary: brush; }`).
+-   **Circular Reference Handling:** Detects dependency cycles between variables. If a cycle is detected within the same collection, it attempts to break the loop using the target's concrete value. Otherwise, or if the loop break fails, it resolves with a default value and logs a warning. Collections involved in cycles force a single-file export.
+-   **Name Sanitization:** Cleans up collection, variable, and mode names to be valid Slint identifiers (e.g., converting spaces and special characters to underscores, handling leading digits).
+-   **Export Options:**
+    -   **Separate Files:** Exports each Figma collection into its own `.slint` file. Automatically generates necessary `import` statements between files if cross-collection references exist (unless a cycle forces single-file mode).
+    -   **Single File:** Combines all collections into a single `design-tokens.slint` file. This is also the fallback if dependency cycles are detected.
+-   **README Generation:** Creates a `README.md` file alongside the export, summarizing exported collections, renamed variables, detected circular references, and warnings.
+
+## Usage
```

Comment:
This section would greatly benefit from screenshots, I think. We're in the *Guide* part of our documentation, a visual guide with screenshots would probably work wonders *and* be a little advertisement :)